locales/US_en.json

```diff
@@ -1,23 +1,60 @@
+	"home": {
+		"header": {
+			"title": "Pretendo",
+			"text": "An open source Nintendo Network replacement service"
+		}
+	},
+	"about": {
+		"caption": "What is this?",
+		"title": "About",
+		"text": "Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the WiiU and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite WiiU and 3DS games to their fullest capacity."
+	},
+	"credits": {
+		"caption": "The people responsible",
+		"title": "Credits",
+		"redducks": "Project owner and lead developer",
+		"caramelkat": "Miiverse research and development",
+		"superwhiskers": "crunch library development",
+		"supermariodabom": "Console and other system research",
+		"rambo6glaz": "Network installer and console research",
+		"mrjvs": "Web development and early eShop research",
+		"kxpler442b": "Web development",
+		"jipfr": "Web development",
+		"kinnay": "None of this would be possible without the research and help done by Kinnay on PRUDP and nex"
+
+	},
+	"footer": {
+		"text": "Copyright 2020 PretendoNetwork - Site designed and developed by mrjvs, ben and jip"
+	},
+	"connecting": {
+		"title": "Connecting",
+		"text": "Pretendo is currently not usable by the general public. When the network becomes public, this will contain steps on connecting to the network"
+	},
+	"cemu": {
+		"title": "Cemu",
+		"text": "Pretendo is currently not usable by the general public. When the network becomes public, this will contain steps on connecting to the network using Cemu"
+	},
+	"faq": {
+		"title": "FAQ"
+		"q1": "What is Pretendo?",
+		"a1": "Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the WiiU and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite WiiU and 3DS games to their fullest capacity",
+		"q2": "How do I use Pretendo?",
+		"a2": "Pretendo is currently not in a state that is ready for public use. However, once it is you will be able to use Pretendo simply by running our homebrew patcher on your console",
+		"q3": "Is there an ETA for XYZ?",
+		"a3": "We do not have an exact ETA for any specific features or the project as a whole. There are many parts to the network, updates release for them individually whenever they are ready (for example updates to the account server are (almost always) independent on other parts of the network, such as the friends server). Due to that we cannot give an exact ETA",
+		"q4": "Does Pretendo work on Cemu/emulators?",
+		"a4": "Pretendo is designed for the WiiU and (eventually) 3DS. At this time the only emulator for these consoles with real online capability is Cemu. Cemu does not officially support custom servers, but it will still be possible to use Pretendo with Cemu
+								<br>
+								See the <a href="/cemu">Cemu</a> page for more details",

```

Comment:
You’re accidentally closing your JSON tag here

Reply:
Fixed!